go.sum

```diff
@@ -58,6 +58,8 @@ github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSs
 github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c=
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
 github.com/docopt/docopt-go v0.0.0-20180111231733-ee0de3bc6815/go.mod h1:WwZ+bS3ebgob9U8Nd0kOddGdZWjyMGR8Wziv+TBNwSE=
+github.com/druid-io/druid-operator v0.0.0-20230223202126-9cec3f2814d3 h1:Sd8295B7rOYBG4ISiZFgU+CW1XNyyMXYC50qODueVgQ=
+github.com/druid-io/druid-operator v0.0.0-20230223202126-9cec3f2814d3/go.mod h1:tbtTlEK57N7cwIF/zZeQBt+yYqdQ8NhtxMbgd6NedwA=
```

Comment:
just wondering whats exactly adding these modules. this repo is deprecated.

Reply:
Fixed, probably leftovers